voicevox_engine/engine_manifest.py

```diff
@@ -94,6 +95,9 @@ class SupportedFeatures(BaseModel):
     manage_library: bool | None = Field(
         title="音声ライブラリのインストール・アンインストール"
     )
+    return_resource_url: bool | None = Field(
+        title="speaker_info・singer_infoのリソースをURLで返す"
+    )
```

Comment:
判断難しいけどこの案内が良さそうに感じました！
（「speaker_info・singer_infoの」を消したほうが良いかと思ったけど、このままが良さそう。）

他が体言止めなので合わせときたいかな、くらいですかね！こうかなぁ。
```suggestion
    return_resource_url: bool | None = Field(
        title="speaker_info・singer_infoのリソースをURLで返送"
    )
```
`return_`ではなく`replace_`でも良いかも。う〜〜〜〜ん。returnかなぁ。